components/Fundraise/FundraiseCard.tsx

```diff
@@ -0,0 +1,273 @@
+import React, { ReactElement, useMemo } from "react";
+import { Fundraise, isFundraiseFulfilled } from "./lib/types";
+import { css, StyleSheet } from "aphrodite";
+import numeral from "numeral";
+import colors from "~/config/themes/colors";
+import { timeTo } from "~/config/utils/dates";
+import CommentAvatars from "../Comment/CommentAvatars";
+import ResearchCoinIcon from "../Icons/ResearchCoinIcon";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCircleCheck } from "@fortawesome/pro-solid-svg-icons";
+import { faClock } from "@fortawesome/pro-regular-svg-icons";
+import FundraiseContributorsModal from "./ContributorsModal";
+import Button from "../Form/Button";
+import FundraiseContributeModal from "./ContributeModal";
+import PermissionNotificationWrapper from "../PermissionNotificationWrapper";
+import ContentBadge from "../ContentBadge";
+import { formatBountyAmount } from "~/config/types/bounty";
+import { breakpoints } from "~/config/themes/screen";
+
+export type FundraiseCardProps = {
+  fundraise: Fundraise;
+
+  showSupportButton?: boolean;
+  onUpdateFundraise?: (fundraise: Fundraise) => void;
+};
+
+const FundraiseCard = ({
+  fundraise: f,
```

Comment:
I realize that `fundraise` is a prop on unified_document and that the parent component `<FeedCard>` is very TS unfriendly. However, for the sake of creating rock solid components on a go-forward basis, we can create a `Fundraise` type + a parser for it. We can then invoke it in `FeedCard` only for that particular property so that `FundraiseCard` and its data are TS friendly. In `DocumentHeader`, you can update the `DocumentMetadata` type to include a `fundraise: Fundraise` prop

Reply:
On second look , it appears you already create a type below but the snake case threw me off. Can you update to camel case for consistency? The nice thing about camelcase is that it is a visual queue that an object literal has been converted to TS and been "dealt with".

Reply:
Ah good point, that's a decent convention to have.